packages/rootage/components/switch.yaml

```
@@ -58,19 +58,31 @@ data:
         label:
           color: $color.fg.neutral
           fontWeight: $font-weight.medium
-      enabled,selected:
-        control:
-          color: $color.bg.brand-solid
       disabled:
         control:
           opacity: 0.38
         label:
           color: $color.fg.disabled
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)